qt/python/mantidqt/widgets/superplot/presenter.py

```diff
@@ -88,6 +89,16 @@ def close(self):
         self._view.close()
         del self._model
 
+    def on_resize(self):
+        """
+        Triggered when the window/dockwidgets is(are) resized.
+        """
+        try:
+            self._canvas.figure.tight_layout()
+        except:
```

Comment:
I believe it should suffice to catch here only a `ValueError` type that was behind the reported error message(s) ('bottom cannot be >= top' for the vertical direction and 'left cannot be >= right' for horizontal direction), and let other types propagate as to not hide other possible issues. What do you think?

Reply:
done